static/modules/entity.js

```diff
@@ -97,7 +98,7 @@ export class Entity {
    * Determines what type of collision will be generated when getCollisionShape
    * is called and collisionShape is undefined.
    */
-  /** @type {"Box"|"Circle"} */
+  /** @type {"Box"|"Circle"|"Beam"} */
```

Comment:
Change this string to Capsule. Also why do we even need collisionType if pretty much everywhere we do `typeof` the collision shape? I brought this up in the collision PR.

Reply:
idk ask @Matt-Puentes

Reply:
CollisionType is used so that when we call getCollisionShape, we know what to generate.